From 1cf2182df5fd6073ad9a26fbc2efb7035087c80d Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Fri, 10 Jun 2022 09:51:07 +0300
Subject: [PATCH] [SW-70527] gaudi2: increase ppboot size to 19kb

As sram bist code increases ppboot size, so increased
the size of ppboot from 18KB to 19KB.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: I1f5c0cef4524cf31e5787a5f15aee38042e7d24f
---

diff --git a/arch/gaudi2/arc/Makefile b/arch/gaudi2/arc/Makefile
index 3c5455c..e8eb7d5 100644
--- a/arch/gaudi2/arc/Makefile
+++ b/arch/gaudi2/arc/Makefile
@@ -1,7 +1,7 @@
 include $(HL_DEV_ARCH_COMMON)/Makefile
 
 FW_SIZE := 0x8000
-PPBOOT_MAX_SIZE := 18432
+PPBOOT_MAX_SIZE := 19456
 PREBOOT_BIN_MAX_SIZE := 130048
 
 LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports.elf
diff --git a/arch/gaudi2/arc/common.h b/arch/gaudi2/arc/common.h
index 391cf85..85d61b8 100644
--- a/arch/gaudi2/arc/common.h
+++ b/arch/gaudi2/arc/common.h
@@ -9,7 +9,7 @@
 #include <bitops.h>
 #include <sizes.h>
 
-#define PPBOOT_SIZE			0x4800		/* 18KB */
+#define PPBOOT_SIZE			0x4C00		/* 19KB */
 #define PREBOOT_SIZE			0x1FC00		/* 127KB */
 
 #define FLASH_EFUSE_OFFSET		(FLASH_SECONDARY_OFFSET + \
